src/symbols.js

```diff
@@ -665,8 +661,6 @@ defineSymbol(math, main, op, "\u2a02", "\\bigotimes");
 defineSymbol(math, main, op, "\u2a01", "\\bigoplus");
 defineSymbol(math, main, op, "\u2a00", "\\bigodot");
 defineSymbol(math, main, op, "\u222e", "\\oint");
-defineSymbol(math, main, op, "\u222f", "\\oiint");
-defineSymbol(math, main, op, "\u2230", "\\oiiint");
```

Comment:
Why did these get removed?

Reply:
@kevinbarabash Currently, there are no glyphs for them, but instead they are created by overlaying SVG ovals onto the glyphs from \iint and \iiint: see https://github.com/KaTeX/KaTeX/pull/1430.

Reply:
Thanks for noticing this and cleaning it up.